examples/library/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"math"
+	"os"
+
+	"fmt"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	_ "github.com/influxdata/flux/builtin"
+	"github.com/influxdata/flux/csv"
+
+	"github.com/influxdata/flux/control"
+	"github.com/influxdata/flux/interpreter"
+	"github.com/influxdata/flux/lang"
+	"github.com/influxdata/flux/parser"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+func main() {
+	t := `import "inputs"
+inputs.fromGenerator(start: now(), stop: now(), count: 5, fn: (n) => 1)
+`
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	defer cancelFunc()
+
+	i := interpreter.NewInterpreter()
+	scope := flux.Prelude()
+
+	astPkg := parser.ParseSource(t)
+	if ast.Check(astPkg) > 0 {
+		panic(ast.GetError(astPkg))
+	}
+
+	semPkg, err := semantic.New(astPkg)
+	if err != nil {
+		panic(err)
+	}
+
+	if _, err := i.Eval(semPkg, scope, flux.StdLib()); err != nil {
+		panic(err)
+	}
+
+	v := scope.Return()
+
+	// Ignore statements that do not return a value
+	if v == nil {
+		return
+	}
+
+	// Check for yield and execute query
+	if v.Type() == flux.TableObjectMonoType {
+		t := v.(*flux.TableObject)
+		now, ok := scope.Lookup("now")
+		if !ok {
+			panic(fmt.Errorf("now option not set"))
+		}
+		nowTime, err := now.Function().Call(nil)
+		if err != nil {
+			panic(err)
+		}
+		spec, err := flux.ToSpec([]values.Value{t}, nowTime.Time().Time())
+		if err != nil {
+			panic(err)
+		}
+		compiler := lang.SpecCompiler{
+			Spec: spec,
+		}
+
+		querier := NewQuerier()
```

Comment:
I think you can use:

```
import "github.com/influxdata/flux/cmd/flux/cmd"

cmd.NewQuerier()
```

Reply:
thanks for the suggestion @affo - changed that